src/utils/storyData.ts

```diff
@@ -0,0 +1,93 @@
+import {
+  Browser,
+  ComparisonResult,
+  TestFieldsFragment,
+  TestResult,
+  TestStatus,
+} from "../gql/graphql";
+
+export const makeBrowserInfo = (key: Browser) => ({
+  id: key,
+  key,
+  name: key.slice(0, 1) + key.slice(1).toLowerCase(),
+  version: "<unknown>",
+});
+
+export const makeViewportInfo = (width: number) => ({
+  id: `_${width}`,
+  name: `${width}px`,
+  width,
+  isDefault: width === 1200,
+});
+
+export const headCapture = {
+  captureImage: {
+    imageUrl: "/B.png",
+  },
+};
+
+export const captureDiff = {
+  diffImage: {
+    imageUrl: "/B-comparison.png",
+  },
+};
```

Comment:
I assume this all is some part of a `Build` or `Test` interface?
Should we actually use the correct type here?

Reply:
I'm not really sure I just moved this code out of an existing story file. I'll see what I can dig up.